mautrix/bridge/matrix.py

```diff
@@ -328,14 +347,18 @@ async def handle_message(self, room_id: RoomID, user_id: UserID, message: Messag
             if await self.allow_bridging_message(sender, portal):
                 await portal.handle_matrix_message(sender, message, event_id)
             else:
-                self.log.trace("Ignoring event %s from %s: not allowed to send to portal",
-                               event_id, sender.mxid)
+                await bail(
+                    f"Ignoring event {event_id} from {sender.mxid}: not allowed to send to portal"
+                )
             return
 
         if message.msgtype != MessageType.TEXT:
+            await bail(f"Event {event_id} is not m.text. Ignoring.")
             return
         elif not await self.allow_command(sender):
-            self.log.trace("Ignoring command %s from %s", event_id, sender.mxid)
+            await bail(
+                f"Ignoring command {event_id} from {sender.mxid}: not allowed to perform command"
```

Comment:
It looks like this will bail with REMOTE step, but since it was never intended to go remote should it have the COMMAND step?